tools/single_use/Normalize_Consent_Data.php

```diff
@@ -1,22 +1,22 @@
 <?php
 /**
- * This script is written for a one time use only to facilitate the normalization of the consent feature of candidate parameters. 
- * 
+ * This script is written for a one time use only to facilitate the normalization of the consent feature of candidate parameters.
```

Comment:
not rebased properly?

Reply:
@zaliqarosli I think it's alright, accidentally modified it and undid the changes. Only difference now is my IDE cleaned some extra spaces.